src/nuxt.config.js

```diff
@@ -238,6 +238,8 @@ const nuxtConfig = {
           'https://substackapi.com',
           '*.crisp.chat',
           'connect.facebook.net',
+          'https://www.youtube.com/iframe_api',
+          'https://www.youtube.com/s/player/d23221b6/www-widgetapi.vflset/www-widgetapi.js',
```

Comment:
Why do we need these script?

Reply:
Because the embed calls these APIs, and one of the reasons for the broken display is also due to this.

<img width="588" alt="截圖 2023-12-17 上午10 29 19" src="https://github.com/likecoin/liker-land/assets/75730405/59fb9f40-1001-4d2d-a75a-1e9594567f80">
<img width="826" alt="截圖 2023-12-17 上午10 29 37" src="https://github.com/likecoin/liker-land/assets/75730405/d9df4c39-7afa-46e5-83a1-9711caabe9bb">